html/php-pre-challenge1/index.php

```diff
@@ -1,5 +1,17 @@
 <?php
 for ($i = 1; $i <= 100; $i++) {
     //ここからコードを書く
-    echo $i;
+    if($i % 3 === 0 && $i % 5 === 0){
+        //multiple of 3 and 5
+        echo 'MULTIPLE OF 3 AND 5/　3の倍数であり、５の倍数' . "<br>";
```

Comment:
アルゴリズムは良いです！
サンプル
https://storage.googleapis.com/slite-api-files-production/files/7b0ddd6c-af2d-477b-ae87-09ae0786f7ef/sample.html
と同じ出力結果にしてください。自動でテストしたいので。
英数字の全角と半角にも注意してください。
```suggestion
        echo '3の倍数であり、5の倍数' . "<br>";
```